.github/CODEOWNERS

```
@@ -30,7 +30,7 @@ pulse/                @Qiskit/terra-core @eggerdj @wshanks
 synthesis/            @Qiskit/terra-core @alexanderivrii @ShellyGarion
 scheduler/            @Qiskit/terra-core @eggerdj @wshanks
 visualization/        @Qiskit/terra-core @nonhermitian
-primitives/           @Qiskit/terra-core @ikkoham @t-imamichi
+primitives/           @Qiskit/terra-core @ikkoham @t-imamichi @chriseclectic @ihincks
```

I made a primitives team in the Qiskit org just now to make this a bit easier

```suggestion
primitives/           @Qiskit/terra-core @Qiskit/qiskit-primitives 
```

---

yep, works for me